reference/7.0/Microsoft.PowerShell.Core/About/about_Output_Streams.md

```diff
@@ -0,0 +1,109 @@
+| 4        | **Verbose** stream     | PowerShell 3.0 | `Write-Verbose`     |
+| 5        | **Debug** stream       | PowerShell 3.0 | `Write-Debug`       |
+| 6        | **Information** stream | PowerShell 5.0 | `Write-Information` |
+
+> [!NOTE]
+> There is also a **Progress** stream in PowerShell. Use `Write-Progress`
+> cmdlet to write to this stream. This stream does not support redirection.
+
+## Success stream
+
+The **Success** stream is the default stream for normal, successful results.
+Use the `Write-Output` cmdlet to explicitly write objects to this stream. This
+is the stream used for transporting objects through the PowerShell pipeline.
+The **Success** stream is connected to the **stdout** stream for native
+applications.
+
+## Error stream
+
+The **Error** stream is the default stream for error results. Use the
+`Write-Error` cmdlet to explicitly write to this stream. The **Error** stream
+is connected to the **stderr** stream for native applications. Under most
+conditions, these errors can terminate the execution pipeline. Errors written
+to this stream are also added the the `$Error` automatic variable. For more
+information, see [about_Automatic_Variables](about_Automatic_Variables.md).
+
+## Warning stream
+
+The **Warning** stream is intended for error conditions that are less severe
+than errors written to the **Error** stream. Under normal conditions, these
+warnings do not terminate execution. Warnings are not written to the `$Error`
+automatic variable. Use the `Write-Warning` cmdlet to explicitly write to this
+stream.
+
+## Verbose stream
+
+The **Verbose** stream is intended for messages that help users troubleshoot
+command as they are run interactively or from a script. Use the `Write-Verbose`
+cmdlet to explicitly write messages to this stream. Many cmdlets provide
+verbose output that useful for understanding internal works of the cmdlet. The
+verbose messages are output only when you use the `-Verbose` common parameter.
+For more information, see [about_CommonParameters](about_CommonParameters.md).
+
+## Debug stream
+
+The **Debug** stream is for messages that scripters understand why their
+command is failing. Use the `Write-Debug` cmdlet to explicitly write to this
+stream. The debug messages are output only when you use the `-Debug` common
+parameter. For more information, see
+[about_CommonParameters](about_CommonParameters.md).
+
+Debug messages are intended for script and cmdlet developers more than end
+users. These debug messages can contain internal details necessary for deep
+troubleshooting.
+
+## Information stream
+
+The **Information** stream is intended to provide message that help a user
+understand what a script is doing. It can also be used by developers as an
+additional streams used to pass information through PowerShell. The developer
+can tag stream data and have specific handling for that stream. Use the
+`Write-Information` cmdlet to explicitly write to this stream.
+
+## Progress stream
```

Comment:
Since this gets an H2, I'm pretty sure the above table should include progress in table instead of as a note.

Reply:
Fixed